src/bin/firebase.js

```diff
@@ -23,6 +23,7 @@ updateNotifier.notify({ defer: true, isGlobal: true });
 var client = require("..");
 var errorOut = require("../errorOut").errorOut;
 var winston = require("winston");
+var { SPLAT } = require("triple-beam");
```

Comment:
`triple-beam` should be added as a direct dependency if we are requiring here (`npm` will de-duplicate it for us)

Reply:
okie will do.